.gitignore

```diff
@@ -1,6 +1,6 @@
 .DS_Store
 *lastfailed
-*coverage
```

Comment:
Just remove the line instead of commenting it out.

Reply:
Done